index.bs

```diff
@@ -1696,22 +1696,80 @@ Team Representative in an Interest Group</h5>
 <h2 id=group-lifecyle>
 Lifecycle of Chartered Groups</h2>
 
-<h3 id="WGCharterDevelopment">
-Initiating Charter Development</h3>
-
-	W3C creates charters for [=chartered groups=]
-	based on interest from the Members and Team.
-	The Team <em class="rfc2119">must</em> notify the Advisory Committee
-	when a charter for a new Working Group or Interest Group is in development.
-	This is intended to raise awareness,
-	even if no formal proposal is yet available.
-	Advisory Committee representatives <em class="rfc2119">may</em> provide
-	feedback on the <a href="#ACCommunication">Advisory Committee discussion list</a>
-	or via other designated channels.
-
-	W3C <em class="rfc2119">may</em> begin work
-	on a Working Group or Interest Group charter
-	at any time.
+	W3C creates [=charters=] for [=chartered groups=] based on interest from the [=Members=] and [=Team=].
+	[=Charters=] are formally approved through an [=AC Review=] and [=W3C Decision=] (see [[#CharterReview]]).
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+	The [=Team=] <em class=rfc2119>may</em> send notice to the [=AC=]
```

Comment:
why not MUST?

Reply:
Because, as we have a template for charters, or in the case of recharterings, in straightforward cases, there may barely be any time between the Team thinking that it should start working on a charter, and that charter being ready for formal charter refinement. In which case it's a little silly for the team to send two mails in a row to the whole AC.